www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -48,6 +51,10 @@ const useStyles = makeStyles((theme) => ({
   autocompletePopper: {
     zIndex: theme.zIndex.tooltip,
   },
+  clearFilter: {
+    padding: 'revert',
+    position: 'inherit',
+  },
```

Comment:
Why applying this style ? What happens if you remove this ?

Reply:
Without application style  they look like that: 
<img width="493" alt="Capture d’écran 2021-09-16 à 15 43 16" src="https://user-images.githubusercontent.com/16045498/133623183-0d0f6873-e351-4342-b5f9-69a65761f378.png">


With style clearFilter:
<img width="493" alt="Capture d’écran 2021-09-16 à 15 41 38" src="https://user-images.githubusercontent.com/16045498/133622965-fb62902d-6b3d-4606-b9e8-246c9529f6ff.png">

In two case it's not perfect :/

Reply:
> Without application style they look like that:
> <img alt="Capture d’écran 2021-09-16 à 15 43 16" width="493" src="https://user-images.githubusercontent.com/16045498/133623183-0d0f6873-e351-4342-b5f9-69a65761f378.png">
> 
> With style clearFilter:
> <img alt="Capture d’écran 2021-09-16 à 15 41 38" width="493" src="https://user-images.githubusercontent.com/16045498/133622965-fb62902d-6b3d-4606-b9e8-246c9529f6ff.png">
> 
> In two case it's not perfect :/ but it's works , that's something ^^

Reply:
BY closing PR on Frontend it's works as expected